### PR TITLE
Remove therealssh

### LINKED ISSUE #55: Remove therealssh from codebase
**Describe the bug**
Dmsgpty suppprt was recently added and there does not seem to be a point in keeping therealssh around. It seems difficult enough to maintain to warrant being removed.

### PR DESCRIPTION
**Pull request by nkryuchkov**
Fixes #55 

 Changes:	
- Remove `therealssh` from the codebase

How to test this PR:
- `make test`

### CODE REVIEW

**Comment by nkryuchkov** [comment]
Should be merged simultaneously with https://github.com/SkycoinPro/skywire-services/pull/44

File: pkg/restart/restart_test.go [hunk 1]
```diff
@@ -50,23 +50,32 @@ func TestContext_Start(t *testing.T) {
 		cmd := "bad_command"
 		cc.cmd = exec.Command(cmd) // nolint:gosec
 
-		// TODO(nkryuchkov): Check if it works on Linux and Windows, if not then change the error text.
-		assert.EqualError(t, cc.Start(), `exec: "bad_command": executable file not found in $PATH`)
+		// TODO(nkryuchkov): Add error text for Windows
```
Comment (jdknives) [inline pkg/restart/restart_test.go:53]: Not related to this PR per se, but we do not need to check for windows compatibility.

**Review by jdknives — APPROVED** [review]
Looks good to me.

File: pkg/restart/restart_test.go (hunk 1 above)
Comment (nkryuchkov) [reply pkg/restart/restart_test.go:53]: @jdknives I'm going to remove it

**Pull request merged**